charmtools/build/inspector.py

```diff
@@ -5,12 +5,19 @@
 from charmtools import utils
 
 theme = {
-    0: "normal",
```

Comment:
there's a few references to `theme.get(x, 'normal')` below.  I think you'll need to either `s/white/normal` here or update those refs below.

Reply:
I think those are correct.  `normal` just means "go back to default styling" which might not actually be `white` depending on the user's term config.

Reply:
Ah, but to your point, if a light theme, explicitly using `white` won't render well.